src/main/java/io/cryostat/recordings/RecordingsModule.java

```diff
@@ -87,6 +87,12 @@ static long provideJmxConnectionTimeoutSeconds(Environment env) {
         return Math.max(1, Long.parseLong(env.getEnv(Variables.JMX_CONNECTION_TIMEOUT, "3")));
     }
 
+    @Provides
+    @Named(Variables.PUSH_MAX_FILES_ENV)
+    static String providePushMaxFiles(Environment env) {
```

Comment:
The `Integer.parseInt` could be done here so that the sites where the value is injected can just use it as-is.

Reply:
Oh I see, the injection site actually uses it as a String for a default parameter on the request attributes map. I think it still makes conceptual sense for this value to be converted and injected as an Integer here, and then if the callsite needs it in String form then it can do that conversion...